draft-packages/avatar-group/KaizenDraft/AvatarGroup/AvatarGroup.spec.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react"
+import { render, screen } from "@testing-library/react"
+import { AvatarGroup } from "./AvatarGroup"
+
+const imgAvatars = [
+  {
+    fullName: "Adirana Aniseed",
+    avatarSrc:
+      "https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png",
+    size: "medium",
+  },
+  {
+    fullName: "Adirana Aniseed",
+    avatarSrc:
+      "https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png",
+    size: "medium",
+  },
+  {
+    fullName: "Adirana Aniseed",
+    avatarSrc:
+      "https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png",
+    size: "medium",
+  },
+  {
+    fullName: "Adirana Aniseed",
+    avatarSrc:
+      "https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png",
+    size: "medium",
+  },
+]
+
+const largeAvatars = [
+  {
+    fullName: "Adirana Aniseed",
+    avatarSrc:
+      "https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png",
+    size: "large",
+  },
+  {
+    fullName: "Adirana Aniseed",
+    avatarSrc:
+      "https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png",
+    size: "large",
+  },
+]
+
+describe("<AvatarGroup />", () => {
+  it("renders a list of avatars up to the maxVisible value", () => {
+    render(<AvatarGroup avatars={imgAvatars} maxVisible={4} />)
+    expect(
+      screen.getAllByRole("img", { name: "Adirana Aniseed" })
+    ).toHaveLength(4)
+  })
+  it("Overides the size of individual Avatars with the size set on AvatarGroup", () => {
+    const { container } = render(
+      <AvatarGroup avatars={largeAvatars} maxVisible={4} size="small" />
+    )
+    const avatarsWithLargeClass = container.querySelectorAll(".large").length
```

Comment:
Maybe in extension to this overriding test,  check that .small has been added as well and .large removed